lib/chirp/scripts/cache

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+main() {
+  # TODO
```

Comment:
Is this PR waiting for this to be implemented, or is the intention to merge this with the TODO?